tests/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+function appengine_deploy()
+{
+    module="$1"
+
+    pushd "${module}"
+
+    mvn clean appengine:deploy verify \
+      -Dapp.deploy.project="${gcloud_projectid}" \
+      -Dapp.deploy.version="${test_runid}" \
+      -Dapp.deploy.promote=false
+
+    ret=$?
+
+    popd
+
+    return ${ret}
+}
+
+gcloud info
+gcloud auth list
+
+gcloud_projectid=$(gcloud info | sed -rn 's/Project: \[(.*)\]/\1/p')
+test_runid=$(date +%Y%m%d-%H%M%S)
+
+echo "GCloud Project: $gcloud_projectid"
+
+# Push the as-built jetty9:latest to gcr.io/${gcloud_projectid}/jetty9:testing"
+
+docker tag "jetty9:latest" "gcr.io/${gcloud_projectid}/jetty9:testing"
+gcloud docker push "gcr.io/${gcloud_projectid}/jetty9:testing"
+
+# Deploy and Execute the individual test-war-* tests
+
+for module in test-war-*
+do
+  echo "module: $module"
+  appengine_deploy "${module}"
+done
```

Comment:
True, though if you just wanted to test that the jetty runtime obeys the `GAE_MEMORY_MB` env variable, you could just override that variable with an arbitrary value in a custom dockerfile, and then verify that the jvm was started with max heap set to 80% of that value. 

I don't think we need to worry about testing that the `GAE_MEMORY_MB` value is set properly for various VM configurations; I would consider that to be on the app engine platform's side of the contract. @meltsufin Do you have any thoughts on this?